### DISCUSSION TITLE
Help！How to Add ICP Record Information to the Dify Home Page?

### DISCUSSION THREAD

**Discussion by fisherever** [langgenius/dify#11485] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I tried to add the ICP record information from the Ministry of Industry and Information Technology of China to the Dify home page, and I modified the code in two places. However, it did not take effect after recompiling in Docker.

### 2. Additional context or comments

- 1st try, edit /home/lighthouse/dify/web/app/signin/layout.tsx, after the very "All rights reserved"
```html
        </div>
        <div className='px-8 py-6 system-xs-regular text-text-tertiary'>
          © {new Date().getFullYear()} LangGenius, Inc. All rights reserved.
          {/* 添加备案信息链接 */}
          <a href="http://www.miitbeian.gov.cn" target="_blank" rel="noopener noreferrer" className="mt-2 block text-blue-500 hover:underline">
            备案号：粤ICP备号-1
          </a>
        </div>
```

- 2nd try, edit /home/lighthouse/dify/web/app/signin/normalForm.tsx
```html
<div className="w-full block mt-2 system-xs-regular text-text-tertiary">
  {t('login.tosDesc')}
  &nbsp;
  <Link
    className='system-xs-medium text-text-secondary hover:underline'
    target='_blank' rel='noopener noreferrer'
    href='https://dify.ai/terms'
  >{t('login.tos')}</Link>
  &nbsp;&nbsp;
  <Link
    className='system-xs-medium text-text-secondary hover:underline'
    target='_blank' rel='noopener noreferrer'
    href='https://dify.ai/privacy'
  >{t('login.pp')}</Link>
  &nbsp;&nbsp;
  <a
    className='system-xs-medium text-text-secondary hover:underline'
    target='_blank' rel='noopener noreferrer'
    href='https://beian.miit.gov.cn/'
  >备案号：粤ICP备号-1</a>
</div>
```

**Comment by wjj2301717059** [comment]
have been solved？i meet same problem

**Comment by CarbonPool** [comment]
Put dify into iframe
